master-qa-199845

Songyuewen opened the pull request:
https://liferay.atlassian.net/browse/LPS-199845

Commit:
LPS-190135 Create Client Extensions

Commit:
LPS-190133 Update Request Status To Awaiting Organization Review

Commit:
LPS-190132 Update Request Status When Organization Is Review

manuferreira committed:
LRAC-14444 Update delete ab test macro to cover the redirect deletion

manuferreira committed:
LRAC-14444 Add macro to assert the AB Test empty state on DXP side

manuferreira committed:
LRAC-14444 Add xpath to track the modal content

manuferreira committed:
LRAC-14444 Add ABTestDraftDeleteButtonRedirectsToDXP testcase

manuferreira committed:
LRAC-14444 Add ABTestDraftReviewButtonRedirectsToDXP testcase

manuferreira committed:
LRAC-14444 Update echo tests to include the newly created macros

manuferreira committed:
LRAC-14444 SF

Commit:
LPS-181776 Standardize

Commit:
LRAC-14542 Remove the macro that uses a functionality no longer present in the product

Commit:
LRAC-14542 Rename DELETE_CONFIRMATION_BUTTON to CONFIRMATION_BUTTON path

Commit:
LRAC-14542 Remove deleteABTestInAC macro

Commit:
LRAC-14542 Rename statusABTestInAC to statusABTestInDXP macro

Commit:
LRAC-14542 Improve the condition present within the deleteABTest macro

Commit:
LRAC-14542 Add tearDownABTest macro

Commit:
LRAC-14542 Ignore all tests that created more than one variant

Commit:
LRAC-14542 Split the ABTest.testcase file into ABTestACSide.testcase and ABTestDXPSide.testcase

Commit:
LRAC-14542 Move part of the code from each test to setUp

Commit:
LRAC-14542 Update tearDown to delete the AB test before disconnecting

Commit:
LRAC-14542 Add CheckTerminateButtonRedirectingToDXP testcase

Commit:
LRAC-14542 SF

Commit:
LRAC-14543 Update terminateTest macro

Commit:
LRAC-14543 Add viewABTestStatusInACList macro

Commit:
LRAC-14543 Update setUp and tearDown

Commit:
LRAC-14543 Add CanTerminateABTestAndCreateNewOne testcase

Commit:
LRAC-14543 SF

Commit:
LRAC-14544 Update NAME_LIST path

Commit:
LRAC-14544 Add CanDeleteTerminatedTestOnDXP testcase

Commit:
LRAC-14544 SF

Commit:
LRAC-14545 Update deleteABTest macro to use the new functionality that redirects to DXP

Commit:
LRAC-14545 Update terminateABTest macro to use the new functionality that redirects to DXP

Commit:
LRAC-14545 Move viewABTestStatusInACList macro from ABTest.macro to ACTests.macro

Commit:
LRAC-14545 Update openABTestInAC macro

Commit:
LRAC-14545 Add viewVariantImprovementNumberInDXP macro

Commit:
LRAC-14545 Add getVariantImprovementNumberInDXP macro

Commit:
LRAC-14545 Add viewVariantImprovementNumberInAC macro

Commit:
LRAC-14545 Add getVariantImprovementNumberInAC macro

Commit:
LRAC-14545 Add viewABTestStatusSummary macro

Commit:
LRAC-14545 Add viewABTestResultAlertMessage macro

Commit:
LRAC-14545 Fix ANALYTICS_CLOUD_LOGIN_BUTTON path

Commit:
LRAC-14545 Add CheckABTestSidePanelWhenNoDeclaredWinner testcase

Commit:
LRAC-14545 SF

Commit:
LRAC-14545 Update IMPROVEMENT_NUMBER path

Commit:
LRAC-14545 Remove CanCreateStartABTest and CanDeleteABTest test from quarantine

Commit:
LRAC-14545 Fix the task text

manuferreira committed:
LRAC-14573 Add xpath to publish one of the variants

manuferreira committed:
LRAC-14573 Add xpath related to the AB test panel

manuferreira committed:
LRAC-14573 Improve createABTest macro to be able to create a test after a terminated test

manuferreira committed:
LRAC-14573 Add macro to confirm the a terminated test

manuferreira committed:
LRAC-14573 Add macro to publish one of the variants

manuferreira committed:
LRAC-14573 Add macro to delete an AB test only on AC side

manuferreira committed:
LRAC-14573 Add CanPublishVariantFromTerminatedTestAndDeleteTest testcase

manuferreira committed:
LRAC-14574 Add CanPublishControlFromTerminatedTest testcase

manuferreira committed:
LRAC-14575 Add CanPublishControlFromTerminatedTest testcase

manuferreira committed:
LRAC-14573 Update the name of the macro to a better fit

manuferreira committed:
LRAC-14573 SF

brianchandotcom committed:
LPS-181776 Wrong place

brianchandotcom committed:
LPS-181776 Use latest

brianchandotcom committed:
LPS-181776 Match latest pattern

brianchandotcom committed:
LPS-181776 Log should always be the same as the class

brianchandotcom committed:
LPS-181776 Rename

Commit:
LPS-196062 Test delete system ObjectAction

Signed-off-by: Guilherme Sa <guilherme.sa@liferay.com>

Commit:
LPS-196062 Test update system ObjectAction

Signed-off-by: Guilherme Sa <guilherme.sa@liferay.com>

Commit:
LPS-181776 Remove nesting

Commit:
LPS-196062 Test add system ObjectActions

Signed-off-by: Guilherme Sa <guilherme.sa@liferay.com>

rafaprax committed:
LPS-105172 Implement the user SCIM unprovisioning

manuferreira committed:
LRAC-14573 Update CanPublishVariantFromTerminatedTestAndDeleteTest to cover the LRAC-14634 bug

manuferreira committed:
LRAC-14573 Update CanPublishControlFromTerminatedTest to include the edit of a published control and assertion of the AB Test on AC side

thiagobuarqque committed:
LPS-198527 add getOnClickJSModule to allow a way of providing a JS module to be imported at runtime

# breaking_change_report

## What modules/apps/product-navigation/product-navigation-personal-menu-api/src/main/java/com/liferay/product/navigation/personal/menu/PersonalMenuEntry.java

modules/apps/product-navigation/product-navigation-personal-menu-api/src/main/java/com/liferay/product/navigation/personal/menu/PersonalMenuEntry.java now includes a new method getOnClickJSModuleURL();

## Why

Necessary to load a JS module at runtime on Personal Menu entry click.

----

thiagobuarqque committed:
LPS-198528 Add Personal Menu entry 'Accessibility Menu'

thiagobuarqque committed:
LPS-198529 Load JS module on Personal Menu Entry click

manuferreira committed:
LRAC-14573 Update publishVariant macro to assert the success message

manuferreira committed:
LRAC-14573 Remove unecessary code

manuferreira committed:
LRAC-14573 Remove test to be covered by CanTerminateABTestAndCreateNewOne

manuferreira committed:
LRAC-14573 Update test to cover LRAC-14575

manuferreira committed:
LRAC-14573 Move macro to another file

manuferreira committed:
LRAC-14573 Edit the tests that were using the delete macro

manuferreira committed:
LRAC-14573 Move tests to a suitable file

manuferreira committed:
LRAC-14573 SF

Commit:
LPS-198875 upgrade plexus-utils version 3.0.24

thiagobuarqque committed:
LPS-194004 generalize JSOnClickConfig to be passed as the default argument in the dynamically loaded JS module

# breaking_change_report

    ## What modules/apps/product-navigation/product-navigation-personal-menu-api/src/main/java/com/liferay/product/navigation/personal/menu/PersonalMenuEntry.java

    In modules/apps/product-navigation/product-navigation-personal-menu-api/src/main/java/com/liferay/product/navigation/personal/menu/PersonalMenuEntry.java the behavior of getJSOnClickConfigJSONObject has been changed so that it requires providing a Javascript function to getOnClickJSModuleURL that will be called with the value returned by getJSOnClickConfigJSONObject as it's parameter.

    ## Why

    This makes getJSOnClickConfigJSONObject generic so that it can be used for any type of on click interaction, not just for opening a selection modal.

thiagobuarqque committed:
LPS-194004 open "My Sites" from Personal Menu using JS module

danvb112 committed:
LPS-198585 Refactor SingleSelect component to use Clay Picker

danvb112 committed:
LPS-198585 Change all previous SingleSelect implementations to meet new ClayPicker requirements in all object-web and notification-web modules

danvb112 committed:
LPS-198585 Refactor ModalAddObjectValidation to use the useForm hook

danvb112 committed:
LPS-198585 Replace all uses of Select to use SingleSelect

danvb112 committed:
LPS-198585 Replace all uses of SelectWithOptions to use SingleSelect

thiagobuarqque committed:
LPS-194004 baseline

Commit:
LPS-198873 Upgrade antisamy version to 1.7.4 to avoid vulnerability

Commit:
LPS-198874 Upgradle plexus-utils version to 3.0.24 to avoid vulnerability

manuferreira committed:
LRAC-14573 Fix xpath name

jaimelr10 committed:
LPS-200105 Add new read-only field to the schema and to the EntityModel to be used in the filter parameter

amosfong committed:
ISSD-470 add download ticket attachment API

murilostodolni committed:
LPS-199635 Create count method to count how many objectFolder existing in a company

murilostodolni committed:
LPS-199635 buildService

murilostodolni committed:
LPS-199635 If there is more than one objectFolder, show the ActionDropdownItem

robertoDiaz committed:
LPS-199854 Add an option to expire all versions in DL

sergiojimcos committed:
LPS-198518 Update headless-builder-bath-engine json to update group property to site

sergiojimcos committed:
LPS-198518 Change to Site enum value parameter in APIApplication interface

sergiojimcos committed:
LPS-198518 Update to SITE in impl module

sergiojimcos committed:
LPS-198518 Update test cases using the SITE parrameter

sergiojimcos committed:
LPS-198518 Update using site parameter in web module

sergiojimcos committed:
LPS-198518 buildLanguage

sergiojimcos committed:
LPS-198518 Update several missing messages

sergiojimcos committed:
LPS-198518 Add new validation:
- If a Response Schema main object definition ERC is not scoped as its related API Endpoint, it will throw an Exception
- If a Request Schema main object definition ERC is not scoped as its related API Endpoint, it will thrown an Exception.

sergiojimcos committed:
LPS-198518 Update Poshi tests

sergiojimcos committed:
LPS-198518 Pass scope property as argument to use it in some tests and apply them

sergiojimcos committed:
LPS-198518 Add test case for this validation

manuferreira committed:
LRAC-14573 Remove the macro of the wrong file

manuferreira committed:
LRAC-14573 Update the macro to cover the cases including the control and the variant

manuferreira committed:
LRAC-14573 Update the tests to use the correct macro

manuferreira committed:
LRAC-14573 Update the test to do the right deletion and assertion

samuelleolopes committed:
ISSD-1420 added External Reference Code to Customer Portal Picklist

Commit:
LPS-149272 Avoid Zip Slip vulnerability in CSSBuilder

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

Songyuewen commented:
ci:test:sf

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 6 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/5d4ce1ff8a61769f109947c97f6187dbb35cfc28">5d4ce1ff8a61769f109947c97f6187dbb35cfc28</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/Songyuewen/liferay-portal/tree/master-qa-199845">master-qa-199845</a><br/>Branch GIT ID: <a href="https://github.com/Songyuewen/liferay-portal/commit/789f77ab40a85a906a312b4327643eabe36cfdde">789f77ab40a85a906a312b4327643eabe36cfdde</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-15.liferay.com/job/test-portal-source-format/4938/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-15.liferay.com/userContent/jobs/test-portal-source-format/builds/4938/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-15.liferay.com/job/test-portal-source-format/4938/">test-portal-source-format#4938</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-15.liferay.com/userContent/jobs/test-portal-source-format/builds/4938/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/suilin/liferay-portal/pull/685">suilin#685</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2446878716">[master] ci:test:sf - Songyuewen &gt; suilin - PR#685 - 2023-10-26[01:37:27]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-15.liferay.com/job/publish-testray-report/4241/">publish-testray-report#4241</a><br/></div></div>

petershin committed:
LPS-200065 Set default deploy directory to osgi/portal

petershin committed:
LPS-200065 Readme

lfbesada committed:
LPS-200225 Hide all the behavior under the feature flag

Commit:
LPS-198873 Use neko-htmlunit 3.6.0

robertoDiaz committed:
LPS-199854 buildlang

jaimelr10 committed:
LPS-200105 Add test for unique property of an objectField

robertoDiaz committed:
LPS-199854 SF

Mikellorza committed:
LPS-200381 Enable case-sensitive for asset tag finders

Mikellorza committed:
LPS-200381 Remove StringUtil.toLowerCase calls for the asset tag name if the FF is enabled

Commit:
LPS-181776 Changes to requests

Commit:
LPS-192809 Send Request Asynchronously or Synchonous

Mikellorza committed:
LPS-200381 Delete unique clause for finder by tag name because from now on will check the unique from the java logic

Mikellorza committed:
LPS-200381 Change return type to collection for the finder by groupId and name since some database are not case-sensitive, so for that reason we have to return a collection

Mikellorza committed:
LPS-200381 buildService

Mikellorza committed:
LPS-200381 Reimplement fetch and get methods

vicnate5 committed:
LPS-200065 Fix for LPKG builds

Commit:
LPS-198528 - Set order integer to 50 so that items can be added before or after it.

jaimelr10 committed:
LPS-200105 Add isUnique method to util

Mikellorza committed:
LPS-200381 Add upgrade process IX_AA52E757 to upgrade the index modified

Mikellorza committed:
LPS-200381 baseline

nNilton committed:
LPS-192809 simplify

nNilton committed:
LPS-192809 inline

jaimelr10 committed:
LPS-200105 Add unique field in ObjectFieldModelDocumentContributor

jaimelr10 committed:
LPS-200105 buildRest

jaimelr10 committed:
LPS-200105 Add field to DTOConverter to show if the object only allows unique values

Mikellorza committed:
LPS-200381 Continue with the same behavior for some methods if the FF is disabled

nNilton committed:
LPS-192809 inline

joshitagaki committed:
LRAC-13083 Improvement fix

nNilton committed:
LPS-200585 [liferay-sample-workspace] add client-extension.yaml

nNilton committed:
LPS-200585 [liferay-sample-workspace] add site resources content

nNilton committed:
LPS-200585 [liferay-dotcom-workspace] add client-extension.yaml

nNilton committed:
LPS-200585 [liferay-dotcom-workspace] rename assets folder to site-initializer

nNilton committed:
LPS-200585 [liferay-tryitnow-workspace] add client-extension.yaml

nNilton committed:
LPS-200585 [liferay-tryitnow-workspace] move content to site-initializer folder

Commit:
LRQA-83371 Fix Organization macro API

Commit:
LRQA-83371 Add API macros and tasks improvements on tests

Commit:
LPS-192809 Standard

Commit:
LPS-192809 it needs

Commit:
LPS-199637 Add new paths and macro to select dropdown item

Commit:
LPS-199637 Replace clicks with new macro selectDropdownItem

Commit:
LPS-199637 Remove and update some paths

Commit:
LPS-199637 Replace asserts on select clay

Commit:
LPS-199637 Refactor macro selectTitleField with currentOption

Commit:
LPS-199637 Remove unnecessary task

Commit:
LPS-199637 Rename the fieldAttachment

gabrielwas committed:
LPS-196062 Rename comment

Commit:
LPS-199637 Add click with new path using clay

Commit:
LPS-199637 Minor changes

Commit:
LRQA-83374 Add API macros on Setup and Add improvements on tests

pedroliveirav committed:
ISSD-1398 Reorganize navigation menu icons

pedroliveirav committed:
ISSD-1398 Fix logic when selecting a navigation item

manuferreira committed:
LRAC-14573 SF

Mikellorza committed:
LPS-200381 Add feature flag LPS-194362

lfbesada committed:
LPS-200225 Take into account that 0 is a valid classTypeId since it is the infoItemFormVariationKey of the Basic Documents

robertoDiaz committed:
LPS-200359 Allow configure No Cache for documents and make it the default option

# breaking_change_report
## What com.liferay.document.library.web.internal.configuration.CacheControlConfiguration.cacheControl

## Why

Add a new option to configure portal to do not cache documents and make it the default option (in order to improve security)

robertoDiaz committed:
LPS-200359 Build lang

antonio-ortega committed:
LPS-200680 We should rescale the image only in case it's larger than the editor's height, otherwise we are enlarging small images reducing their quality

DiegoHu97 committed:
LPS-200685 Add Site for new child doesn't show the name of the parent

Commit:
LPS-105172 Make the UserManager transactional for updates

thiagobuarqque committed:
LPS-194004 simplify module import, frontend team will handle the ESM modules in the future

Commit:
LRQA-83373 Update EVPRequestForm.testcase

Commit:
LRQA-83373 Update tasks

igor-franca committed:
LPS-200780 Avoid Undefined in Map Return

magjed4289 committed:
LPS-200859 [test-fix] Poshi tests

lfbesada committed:
LPS-199784 Populates urlTitle localizations

lfbesada committed:
LPS-199784 Assert that the URL title field is populating the localization as keyword sortable

lfbesada committed:
LPS-199784 SF extract to a method and simplify

lfbesada committed:
LPS-199784 SF join both methods since is the same logic

lfbesada committed:
LPS-199784 SF inline

jkappler committed:
LPS-200860 Only emailFrom shouldn't be included in the FF, since it's validated in the frontend

jkappler committed:
LPS-200860 Adds missing validations

jkappler committed:
LPS-200860 Adds error messages to view. Copied from configuration.jsp, and set embedded to false

jkappler committed:
LPS-200860 Adds navigation param to action URL

ealonso committed:
LPS-200862 Uses FeatureFlagManagerUtil instead a custom configuration

Commit:
LPS-198875 Auto generated

Commit:
ISSD-1554 Fix padding and width for dashboard pages

Commit:
ISSD-1554 Change routes to use orderId

Commit:
ISSD-1554 Create component to label status

Commit:
ISSD-1554 Create hook to get product by the orderId

Commit:
ISSD-1554 Fix padding for dashboard pages

Commit:
ISSD-1554 Change routes to use orderId instead the product

Commit:
ISSD-1554 Fix app details page

Commit:
ISSD-1554 Create AppOutlet to use navbar and header

Commit:
ISSD-1554 Create order details header

Commit:
ISSD-1554 Create getPlacedOrder headless request

Commit:
ISSD-1554 Fix app details page

gaborlovasdotcom committed:
LPS-200898 Temporary disable tests

brianchandotcom committed:
LPS-192809 Update with refresh_other_workspaces.sh

Commit:
LPS-97345 We only ever activate users except when deleting users, which is handled by a separate method

brianchandotcom committed:
LPS-192809 SF

brianchandotcom committed:
LPS-192809 SF

brianchandotcom committed:
LPS-200105 SF

Commit:
LPS-97345 Inactive portal users are considered deleted or non-existent users for SCIM purposes

Commit:
ISSD-1554 Fix padding for dashboards page

brianchandotcom committed:
LPS-200859 SF

Commit:
ISSD-1554 Fix header styles

Commit:
ISSD-1554 Fix order type name logic and add icon to label

Commit:
ISSD-1554 Add alt name for order icon type

Commit:
ISSD-1554 SF

brianchandotcom committed:
LPS-200381 SF/Wordsmith

samuelleolopes committed:
ISSD-1419 Created a field select to record its purpose

brianchandotcom committed:
LPS-194293 Not used?

Commit:
ISSD-1554 Fix manage license jeys route redirection

ealonso committed:
LPS-200882 Update test since the behavior change in LPS-173502 and now we are allowing null values

rafaellajordao committed:
LPS-200955 test fix for ab test variant

Commit:
ISSD-1508 Create Table component

Commit:
ISSD-1508 Add Table to Licenses page

Commit:
ISSD-1508 Updated ddm-template styles

Commit:
LPS-97345 It is a factory configuration

brianchandotcom committed:
LPS-198875 prep next

brianchandotcom committed:
LPS-198875 prep next

brianchandotcom committed:
LPS-198875 prep next

brianchandotcom committed:
LPS-198875 prep next

brianchandotcom committed:
LPS-198875 Regen

brianchandotcom committed:
LPS-198875 prep next

brianchandotcom committed:
LPS-105172 Wordsmith

Commit:
LPS-199635 Baseline

Signed-off-by: Guilherme Sa <guilherme.sa@liferay.com>

joshitagaki committed:
LRAC-13083 SF

Commit:
ISSD-1554 SF

brianchandotcom committed:
LPS-192809 My bad

brianchandotcom committed:
LPS-200065 prep next

brianchandotcom committed:
LPS-200862 SF

brianchandotcom committed:
ISSD-470 SF

ealonso committed:
LPS-200879 Use proper method, since we want to hide it for layout scopes

brianchandotcom committed:
ISSD-470 SF

michaelhashimoto committed:
LRCI-3821 Add modified time to avoid error

rafaellajordao committed:
LPS-195012 Bug automation FilterCategoryFilterByVocabularies

Italo-gabriel committed:
LRQA-83366 Update gherkin tasks for test cases

Italo-gabriel committed:
LRQA-83366 Removing test covered by another scenario

brianchandotcom committed:
LRCI-3821 prep next

Commit:
ISSD-1508 Show order license keys

nNilton committed:
LPS-200585 remove object definitions and add a new fragment

Commit:
ISSD-1508 SF

suilin commented on portal-web/test/functional/com/liferay/portalweb/tests/enduser/publications/PublicationsDM.testcase:
```diff
@@ -213,6 +213,47 @@ definition {
+			key_changeCategory = "Dynamic Data Mapping Structure Layout",
+			key_changeTitle = "Dynamic Data Mapping Structure Layout",
+			locator1 = "PublicationsChanges#DISCARD_CHANGE_BUTTON");
+	}
```
Can you add a step to assert there is no “Edit“ option displayed next to the DDM Structure?

Songyuewen replied:
> Can you add a step to assert there is no “Edit“ option displayed next to the DDM Structure?

Add step assert `PUBLICATIONS_CHANGE_DROPDOWN` element is not present

Songyuewen committed:
LPS-199845 Add test assert deletion Modification Conflict for DDM Structures should not be edited

Songyuewen commented:
ci:test:sf

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 5 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/1602db437455aac488f8ec3d9d599ea43ae8f4dd">1602db437455aac488f8ec3d9d599ea43ae8f4dd</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/Songyuewen/liferay-portal/tree/master-qa-199845">master-qa-199845</a><br/>Branch GIT ID: <a href="https://github.com/Songyuewen/liferay-portal/commit/0ba43423346c103877aeca9a1bc7f40f801e7639">0ba43423346c103877aeca9a1bc7f40f801e7639</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-40.liferay.com/job/test-portal-source-format/3542/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-40.liferay.com/userContent/jobs/test-portal-source-format/builds/3542/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-40.liferay.com/job/test-portal-source-format/3542/">test-portal-source-format#3542</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-40.liferay.com/userContent/jobs/test-portal-source-format/builds/3542/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/suilin/liferay-portal/pull/685">suilin#685</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2453987821">[master] ci:test:sf - Songyuewen &gt; suilin - PR#685 - 2023-11-07[02:28:55]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-40.liferay.com/job/publish-testray-report/13668/">publish-testray-report#13668</a><br/></div></div>

suilin commented:
ci:forward

liferay-continuous-integration commented:
CI is automatically triggering the following test suites:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**relevant**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**stable**

The pull request will automatically be forwarded to the user `brianchandotcom` If the following test suites pass:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**relevant**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**stable**

liferay-continuous-integration commented:
Skipping previously passed test suites:
- `ci:test:relevant`
- `ci:test:sf`
- `ci:test:stable`

liferay-continuous-integration commented:
All required test suite(s) passed.
Forwarding pull request to `brianchandotcom`.
[Console](https://test-1-15.liferay.com/job/forward-pullrequest/542//consoleFull)

liferay-continuous-integration commented:
Pull request has been successfully forwarded to  https://github.com/brianchandotcom/liferay-portal/pull/143439
[Console](https://test-1-15.liferay.com/job/forward-pullrequest/542//consoleFull)
